### PR TITLE
Updated DIN rail clips

### PR DESCRIPTION
**Pull request by th0mpy**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

--Adding more robust din rail mounts for PSU and controller on Voron printers (2.4 confirmed working)

#### How was it tested? 

--Printed several and installed in my working printer. 

#### Any background context you want to provide?

--The din clips as printed from the standard Voron 2.4 has the potential to become unclipped while moving the printer. This is dangerous since it happened a couple times and the 24v PSU fell out. Given that we're dealing with mains this could be a fire risk. I designed these around the stock clips, thicker with an added spring to give better contact with the rail and reduce the potential for anything from becoming unclipped when not desired. The clips still work to easily remove in the future.

### CODE REVIEW

**Review by newphreak — COMMENTED** [review]
Huh, that's  both pretty neat and smart. Good job on the mod.
-In the top level README you need to put your mod in alphabetical order in regards of your username.
-In the mod readme I'd love to see an unaffiliated link with no tracking to buy the springs just for simplicity's sake.
-All filenames need to be renamed to not have spaces and conform with voron naming scheme. (Read the checklist)
-.step files can be combined to a master assembly for simplicity. Just make each version a new component and the component names will follow in the .step
-STL orientation needs to be fixed, it's not printable in it's current orientation. (Just load one and one in the slicer and export plate if you are using superslicer.

**Comment by th0mpy** [comment]
Got it! Thanks for following up. Should be good to go now.

File: printer_mods/README.md [hunk 1]
```diff
@@ -362,6 +362,7 @@ like so:
 | |[Side Fan Support, No Tape](./StvPtrsn/Side_Fan_Support_No_Tape) |Mount the 60mm side fans and inserts without tape |:x: |:x: |:heavy_check_mark: |:x:|
 | |[Parts_Jigs and Mods](./StvPtrsn/Parts_Jigs_and_Mods) |Parts, Jigs and modifications I use for my builds |:heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark:|:heavy_check_mark:|
 | TAKUYA | [WANSVIEW USB Camera Mount](./Takuya/USBCameraMount/) | WANSVIEW 1080p Camera Mount for Voron 2.4 | :x: | :x: | :heavy_check_mark: | :x: |
+| th0mpy | [BEEFY Din Clips](./th0mpy/beefy-din) | Two piece BEEFY Din Clips, holds stronger | :grey_question: | :grey_question: | :heavy_check_mark: | :grey_question: | :grey_question: |
```
Comment (newphreak) [inline printer_mods/README.md:365]: image

**Review by newphreak — CHANGES_REQUESTED** [review]
.step files are usually under a folder named cad, not step.
Other then that and the top-level readme this looks good 👍 So fix those

File: printer_mods/README.md (hunk 1 above)
Comment (th0mpy) [reply printer_mods/README.md]: Thanks for being patient. First one :) 

These should all be fixed now.

**Comment by th0mpy** [comment]
I thought I had that location addressed. I changed the folder and updated the readme. Again, thanks for the patience.

**Review by newphreak — APPROVED** [review]
You're doing great. Looks good to me.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Your mod should go above TiagoBarata's mods in the readme.

**Comment by th0mpy** [comment]
This should be good now.

**Comment by cbc02009** [comment]
Sorry, even though this is 100% not your fault, would you mind moving thesquiffy's mod in the readme too so it's all properly alphabetical? Would just save me having to open a PR to fix it later :)

**Comment by th0mpy** [comment]
Done

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**